js/test/core/season.js

```diff
@@ -102,4 +106,109 @@ define(["globals", "core/season", "util/helpers", "test/helpers"], function (g,
             });
         });
     });
-});
\ No newline at end of file
+
+    describe('core/season playoffs tiebreakers', function() {
+        var tx;
+        before(function(done) {
```

Comment:
`done` is not needed here, you can just return a promise and then it'll automatically realize the test is done when the promise resolves.